v2.0/serial.md

```diff
@@ -16,8 +16,9 @@ The `SERIAL` type is equivalent to [`INT DEFAULT unique_rowid()`](int.html).
 
 In CockroachDB, the following are aliases for `SERIAL`:
 
-- `SMALLSERIAL`
-- `BIGSERIAL`
+Name | Allowed Width | Aliases
```

Comment:
Since `SERIAL` always generates a 64-bit integer, and you never manually do this yourself, instead of listing `Allowed Width` here, I'd suggest we update the intro sentence to:

> The `SERIAL` data type is a column data type that, on insert, generates a 64-bit signed integer from the timestamp and ID of the node executing the insert.

We already state that on the [data type overview page](https://www.cockroachlabs.com/docs/stable/data-types.html). 

Then I think we can go back to a simple bulleted list of aliases. Thoughts, @lhirata?

Reply:
Yeah, that sounds good to me!